readme.md

```diff
@@ -65,7 +79,7 @@ Describe the purpose of views and models in FE JS.
 
 Your Answer:
 ```text
-
+The traditional way of updating a page is for a browser to send a request and receive back an entire page from the backend. A big advantage to front-end JS is that it allows portions of a page to be updated without having to update the entire page. The front-end JS does this by sending requests for the appropriate information to update and then decoding the response back. The front-end model helps parse through these responses in a way that can be used by the front-end view to display and update the page.
 ```
```

Comment:
:+1:

Not only does the view allow us to render the data stored in a model, but it's also responsible for adding functionality that responds to user interaction (e.g., event listeners).